aten/src/ATen/native/UpSample.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
I'd replace this by header guards -- `#pragma once` can have that effect in everything that includes this header.

Reply:
@vfdev-5 Pointed out to me that everywhere in PyTorch does this.